filebeat/module/kibana/log/ingest/pipeline.json

```diff
@@ -0,0 +1,63 @@
+{
+    "description": "Pipeline for parsing Kibana log logs",
```

Comment:
Sorry for spotting it quite late, but could you please remove "log" from the description.

Reply:
done